### DISCUSSION TITLE
analyze-commits: Pull Requests labels instead commit messages

### DISCUSSION THREAD

**Discussion by jkroepke** [semantic-release/semantic-release#3153] [Configuration] [answered]
Hi,

I have a open source monorepo project where I can't guarantee that all commits having the conventional commits message style.

Instead commit messages, I would like to ask if its possible to use pull request label as source, if the next version is major, minor or patch.

I'm already using labels for the [automatic release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) and I would to continue this pattern.

`semantic-release/github` has already some logic to detect pull request based since latest release.

**Comment by travi** [comment, marked as answer]
semantic-release operates on commit messages (those that exist after merging, so actions like squashing can change what those are) and does not factor the PR into the version bump or release notes. 

Also, semantic-release does not provide support for monorepos natively. You will need to leverage a community add-on if you plan to use it in that context.
